core/standalone/README.md

```diff
@@ -380,6 +386,24 @@ This shows an output like below indicating that KubernetesContainerFactory based
 }
 ```
 
+## Launching OpenWhisk standalone with Docker
+
+If you have docker and bash installed, you can launch the standalone openwhisk from the docker image with just:
+
+`bash <(curl -sL https://s.apache.org/openwhisk.sh)`
```

Comment:
I'm getting: No such short-URL; openwhisk.sh

Reply:
May be we skip this for now and later once we have a release done we can create a short url then and update the docs after that.

Reply:
I need the changes to be committed so I can create the short url - it is an egg-and-chicken problem :) once it is created I can do the short url